context/app/markdown/docs/apis.md

```diff
@@ -34,15 +34,9 @@ Examples of Entity endpoints are as follows:
    data for entity
 
 **Figure 3.**
-![An example HuBMAP entity graph consisting of a donor, organ, blocks, tissue slices, data and derived data from a pipeline. In general, a donor and organ are required in the provenance hierarchy where tissue samples (such as blocks and samples) can be organized based on several different tissue sample types.](https://drive.google.com/file/d/14aAyTItvm3teFB5jUX5TVGRpW4oit99b/view?usp=sharing)
+https://drive.google.com/file/d/14aAyTItvm3teFB5jUX5TVGRpW4oit99b/view
```

Comment:
No, we don't want bare URLs sitting on the page. Is there still a permissions problem?